<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistryWidget</class>
 <widget class="QWidget" name="RegistryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="DebitWidget" name="debitTab">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <attribute name="title">
       <string>Приход</string>
      </attribute>
     </widget>
     <widget class="CreditWidget" name="creditTab">
      <attribute name="title">
       <string>Расход</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DebitWidget</class>
   <extends>QWidget</extends>
   <header>debitwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreditWidget</class>
   <extends>QWidget</extends>
   <header>creditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
